test/windowclose.jl

```diff
@@ -11,6 +11,9 @@ window = GLFW.CreateWindow(800, 600, "InexactError")
 ccall( (:glfwSetWindowShouldClose, GLFW.lib), Void, (GLFW.WindowHandle, Cint), window, 189)
 @test GLFW.WindowShouldClose(window)
 
+sleep(0.25)
```

Comment:
No reason. Just wondered if there were events that might need a little time to sync. I'll delete if you prefer.